### DISCUSSION TITLE
Are 'TECHNILINE 43 ISI' supported?

### DISCUSSION THREAD

**Discussion by oelkie** [PredatH0r/ChanSort#406] [General]
Does anyone have this german model?
Technisat didn't give an exact answer about exporting channellists for 'TECHNILINE 43 ISI' and 'TECHNIVISION UHD43A'
I'm missing an detailed list of supported models here.

**Comment by PredatH0r** [comment]
I'm not familiar with any Techniline / Technivision products.

Unfortunately it is impossible to create/maintain a detailed list of supported devices.
Many manufacturers keep inventing new file formats or variations every year, sometimes even multiple when they use different hardware/firmware platforms for different product lines (mostly dependent on price range).
There is also the other case, where multiple brands sell devices under different names that have the same hardware/firmware platform inside.

I can only tell if specific device is supported when I see a file that has been exported by it.
